app/routes/events/components/EventEditor/index.tsx

```diff
@@ -138,13 +138,14 @@ const EventEditor = () => {
   const pools = useAppSelector((state) =>
     selectPoolsWithRegistrationsForEvent(state, { eventId }),
   );
-  const imageGalleryEntries = useAppSelector(selectImageGalleryEntries);
+  const imageGalleryEntries = useAppSelector(selectAllImageGalleryEntries);
   const imageGallery = imageGalleryEntries?.map((image) => ({
     key: image.key,
     cover: image.cover,
     token: image.token,
     coverPlaceholder: image.coverPlaceholder,
   }));
+  console.log(imageGalleryEntries);
```

Comment:
.

Reply:
Ah, shit. I do this too much. I feel like it should be caught by some ESLint rule or something